src/logger/loggerTaskEx.c

```diff
@@ -232,7 +232,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* Prepare a Sample */
-                struct sample *sample = &g_sample_buffer[bufferIndex];
+                struct sample *sample = g_sample_buffer + bufferIndex;
```

Comment:
what are your thoughts on this being better in terms of readability? I tend to think the array reference is clearer to understand.